pkg/graph/schema.graphql

```diff
@@ -1332,6 +1343,27 @@ input PlanCrTdlChanges @goModel(model: "map[string]interface{}") {
     note: String
 }
 
+
+type AuditChange {
```

Comment:
This is something that doesn't have to be added right now but what it be beneficial to include the objects current values here too? Something like `currentFields: Map!`. That may be overkill but just a thought

Reply:
Thanks @rileyanderson , for now, I opted to at least not copy the current values in the DB record besides the fields that were changed. I think it would make it quite a bit more complex (though possible) to include it here at the moment. I'm inclined to leave it for now, and we can add that if a future use case calls for it.